Tools/AP_Periph/Parameters.h

```diff
@@ -38,11 +38,21 @@ class Parameters {
         k_param_msp_port,
         k_param_notify,
         k_param_esc_pwm_type,
+        k_param_can_baudrate2,
+        k_param_can_baudrate3,
+        k_param_can_protocol1,
+        k_param_can_protocol2,
+        k_param_can_protocol3,
     };
 
     AP_Int16 format_version;
     AP_Int16 can_node;
-    AP_Int32 can_baudrate;
+    AP_Int32 can_baudrate1;
```

Comment:
This would require a class, right? Seems overkill.

Reply:
@tridge please take a look at the new array implementation that @bugobliterator helped out with. The commits aren't squashed but this way at least retains his commit credit

Reply:
> This would require a class, right? Seems overkill.

no, just declare like this:
  AP_Int32 can_baudrate[HAL_NUM_CAN_IFACES];
then in param tables use:
 can_baudrate[0]
 can_baudrate[1]
etc
trivial, and makes code much cleaner

Reply:
Yup, done. Please take a look.